### DISCUSSION TITLE
How do you unlock an app after you are done editing it?

### DISCUSSION THREAD

**Discussion by FlaminWrap** [Budibase/budibase#10947] [Help] [answered]
With the new multiplayer collaboration, how do I unlock an app so someone else can edit the design screens after I am done? I know it will time out after 10 mins, but is there a faster way?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @FlaminWrap,

All you need to do is exit the app or close your browser's tab and it will unlock the app/let the next user inline edit the design/automation screens.

The documentation I've linked below will answer your question in a bit more detail;
[Multiplayer collaboration](https://docs.budibase.com/docs/multi-dev-collaboration)

I hope this makes sense and is helpful.
